drf_spectacular/utils.py

```diff
@@ -121,6 +123,11 @@ def is_deprecated(self, path, method):
                     return deprecated
                 return super().is_deprecated(path, method)
 
+            def get_tags(self, path, method):
+                if tags:
```

Comment:
i think ``if tags:`` does not cover all the cases.  that if someone wants no tags. giving `[]` would trigger default handling. 

if think ``if tags is not None:`` would make more sense here, right?